clients/iOS/Grapevine/Grapevine/CalendarViewController.swift

```diff
@@ -40,30 +41,25 @@ class CalendarViewController: UIViewController, CVCalendarViewDelegate, CVCalend
     }
     
     
-    
-    @IBAction func todayPressed(){
-        
-    }
-    
-    @IBAction func weekMonthPressed(sender: UIBarButtonItem){
-        let monthStr = "Month"
-        let weekStr = "Week"
-        if sender.title == weekStr {
-            calendarView.calendarMode = .WeekView
-            sender.title = monthStr
+    @IBAction func changeView(sender: UIBarButtonItem){
+        if self.inMonthView {
+            self.toWeekView()
+            sender.title = "Month"
```

Comment:
I might be confused but shouldn't toWeekView have sender.title = "Week". I think you may have just flipped them?